torchvision/models/quantization/mobilenetv3.py

```diff
@@ -107,13 +110,13 @@ def _mobilenet_v3_model(
         torch.quantization.prepare_qat(model, inplace=True)
 
         if pretrained:
-            _load_weights(arch, model, quant_model_urls.get(arch + '_' + backend, None), progress)
+            _load_weights(arch, model, quant_model_urls.get(arch + '_' + backend, None), progress, False)
```

Comment:
Earlier versions of the SqueezeExcite class used `F.adaptive_avg_pool2d()` and `F.hardsigmoid()` instead of their `nn.Module` equivalents. Using the latter are advised as because QAT can further optimize them.

Loading the old weights, is still possible but the QAT bits of the above two layers will be missing. Passing `strict=false` allows us to use the previous weights and achieve the same accuracy.